internal/admission/validation/ingress/ingress_test.go

```diff
@@ -56,6 +57,7 @@ func TestValidateIngress(t *testing.T) {
 				tt.ingress,
 				logger,
 				fakestore,
+				fake.NewClientBuilder().Build(),
```

Comment:
Nit: this seems to be the same as `fake.NewFakeClient()`.

Reply:
Resolved in https://github.com/Kong/kubernetes-ingress-controller/pull/5979 💪